.github/workflows/ci.yml

```diff
@@ -372,10 +375,24 @@ jobs:
       - name: Run rpcauth test
         run: py -3 test/util/rpcauth-test.py
 
+      - name: Download previous releases
+        if: env.DOWNLOAD_PREVIOUS_RELEASES == 'true'
+        shell: bash
+        run: |
+          mkdir -p ${{ env.PREVIOUS_RELEASES_DIR }}
+          python test/get_previous_releases.py -b -t "${{ env.PREVIOUS_RELEASES_DIR }}"
+
+      - name: Run wallet_migration.py test
+        if: env.DOWNLOAD_PREVIOUS_RELEASES == 'true'
+        env:
+          PYTHONIOENCODING: utf-8
+        run: py -3 test/functional/wallet_migration.py --previous-releases
+
```

Comment:
Why does `wallet_migration.py` need to be run separately, and none of the other tests that require previous releases? These can all be run at the same time from the `test_runner.py` invocation if the `PREVIOUS_RELEASES_DIR` environment variable is set.